packages/addons-v5/test/commands/addons.--all.js

```diff
@@ -27,11 +27,11 @@ describe('addons --all', function () {
     it('prints add-ons in a table', function () {
       return cmd.run({ flags: {} }).then(function () {
         util.expectOutput(cli.stdout,
-          `Owning App    Add-on     Plan                         Price      State
-────────────  ─────────  ───────────────────────────  ─────────  ────────
-acme-inc-api  api-redis  heroku-redis:premium-2       $60/month  created
-acme-inc-www  www-db     heroku-postgresql:hobby-dev  free       created
-acme-inc-www  www-redis  heroku-redis:premium-2       $60/month  creating`)
+          `Owning App    Add-on     Plan                       Price      State
+────────────  ─────────  ─────────────────────────  ─────────  ────────
+acme-inc-api  api-redis  heroku-redis:premium-2     $60/month  created
+acme-inc-www  www-db     heroku-postgresql:no-free  free       created
```

Comment:
Why do we use `mini` in some places and `no-free` in others? They are both replacing `hobby-dev`.
The plan: no-free, cost:free is confusing. Just checking that this is what is desired.

Reply:
I think this `no-free` naming was mentioned in a [previous pr](https://github.com/heroku/cli/pull/2041#discussion_r960044377) that is now outdated. Using mini now would clear confusion. 👍